src/libguac/guacamole/client.h

```diff
@@ -30,7 +30,7 @@
 #include "client-types.h"
 #include "client-constants.h"
 #include "layer-types.h"
-#include "reentrant-rwlock.h"
+#include "guacamole/rwlock.h"
```

Comment:
This should just be:

```c
#include "rwlock.h"
```

within files that are already within the `guacamole/` include path.